mono/utils/mono-threads.h

```diff
@@ -73,6 +73,18 @@ typedef gsize (*MonoThreadStart)(gpointer);
 
 #endif /* #ifdef HOST_WIN32 */
 
+#ifdef ENABLE_NETCORE
+// Inline with CoreCLR heuristics, https://github.com/dotnet/runtime/blob/385b4d4296f9c5cb82363565aa210a1a37f92d90/src/coreclr/src/vm/threads.cpp#L6344.
+// Minimum stack size should be sufficient to allow a typical non-recursive call chain to execute,
+// including potential exception handling and garbage collection. Used for probing for available
+// stack space through RuntimeHelpers.EnsureSufficientExecutionStack.
+#if TARGET_SIZEOF_VOID_P == 8
+#define MONO_MIN_EXECUTION_STACK_SIZE 128 * 1024
+#else
+#define MONO_MIN_EXECUTION_STACK_SIZE 64 * 1024
+#endif
```

Comment:
Need parens around it.
Also `#if` is not needed.
And this does not have to be in .h, can be in .c.